hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/contracts/execution/HederaEvmTxProcessor.java

```diff
@@ -179,8 +180,8 @@ public HederaEvmTransactionProcessingResult execute(
         }
     }
 
-    public void setupFields(final boolean contractCreation) {
-        this.intrinsicGas = gasCalculator.transactionIntrinsicGasCost(Bytes.EMPTY, contractCreation);
+    public void setupFields(final Bytes payload, final boolean contractCreation) {
+        this.intrinsicGas = gasCalculator.transactionIntrinsicGasCost(payload, contractCreation);
```

Comment:
Where is `pay-load` defined?

Reply:
It's extracted from the transaction object and forwarded by EvmTxProcessor.

Reply:
Ignore, I was asking why it was `pay-load` vs `payload`.
I couldn't find the variable `pay-load`, turns out the hyphen was just GitHub line wrapping.